### ISSUE TITLE
Not working in Django

### ISSUE THREAD

**Issue by guilherme-funchal** [Atrox/sweetify-django#16]
Hi, I followed all the steps to install on Django and I was not successful.

Inside my views.py I inserted : 

import sweetify

sweetify.fire('Good job!', 'You clicked the button!', 'success')

No message appears.

**Comment by Atrox** [comment]
Please take a look at issue #11. Is sweetalert installed correctly?

**Comment by guilherme-funchal** [comment]
Hello, thank for you reply.

It includes installation in the generation of the docker container and apparently everything is OK.

**Comment by Atrox** [comment]
Did you try this https://github.com/Atrox/sweetify-django/issues/11#issuecomment-562600746?

**Comment by guilherme-funchal** [comment]
I need to install the sweetify2 or just sweetify (pip install sweetify) ?

**Comment by guilherme-funchal** [comment]
In my code I inserted:
sweetify.sweetalert(request, 'Title', button=True)

Error : 
The view aries_community.views.handle_credential_offer didn't return an HttpResponse object. It returned None instead.

**Comment by Atrox** [comment]
As stated in the README "Note: This package does not provide the client-side files of SweetAlert. You have to provide them yourself." you have to install the client-side part yourself. So you need to install both sweetify and [sweetalert2](https://sweetalert2.github.io/).

Regarding your error: Did you read the README at all? You still have to redirect at the end.

**Comment by guilherme-funchal** [comment]
Take a look

                sweetify.success(request, 'test')
                return redirect('/connections')

Nothing happens...

**Comment by Atrox** [comment]
> Did you try this https://github.com/Atrox/sweetify-django/issues/11#issuecomment-562600746?

Did you try this?

**Comment by guilherme-funchal** [comment]
The Django is running inside a container, how do I do this?

**Comment by Atrox** [comment]
This has nothing to do with your container. Open your browser, navigate to your site and execute the line?

**Comment by guilherme-funchal** [comment]
Page not found (404)
Request Method:	GET
Request URL:	http://localhost:8000/Swal.fire('Good%20job!',%20'You%20clicked%20the%20button!',%20'success')
Using the URLconf defined in aries_community_demo.urls, Django tried these URL patterns, in this order:

admin/
accounts/
signup/ [name='signup']
org_signup/ [name='org_signup']
send_invitation/ [name='send_invitation']
update_user/ [name='update_user']
send_invitation_org/ [name='send_invitation_org']
list_connections/ [name='list_connections']
connection_response/ [name='connection_response']
check_connection/ [name='check_connection']
invitation/<token> [name='connection_qr']
form_response/ [name='form_response']
list_conversations/ [name='list_conversations']
select_credential_offer/ [name='select_credential_offer']
select_credential_proposal/ [name='select_credential_proposal']
credential_offer/ [name='credential_offer']
credential_proposal/ [name='credential_proposal']
cred_revoke/ [name='cred_revoke']
send_message/ [name='send_message']
cred_offer_response/ [name='cred_offer_response']
cred_proposal_response/ [name='cred_proposal_response']
message_show/ [name='message_show']
message_remove/ [name='message_remove']
cred_proposal_show/ [name='cred_proposal_show']
cred_proposal_delete/ [name='cred_proposal_delete']
remove_connection/ [name='remove_connection']
select_proof_request/ [name='select_proof_request']
send_proof_request/ [name='send_proof_request']
proof_req_response/ [name='proof_req_response']
proof_select_claims/ [name='proof_select_claims']
view_proof/ [name='view_proof']
profile/ [name='aries_profile']
data/ [name='aries_data']
wallet/ [name='aries_wallet']
connections/ [name='connections']
conversations/ [name='conversations']
credentials/ [name='credentials']
remove_credentials/ [name='remove_credentials']
revoke_credentials/ [name='revoke_credentials']
agent_cb/<cb_key>/topic/<topic>/ [name='agent_callback']
view_dashboard/ [name='view_dashboard']
[name='login']
^static\/(?P<path>.*)$
^media\/(?P<path>.*)$
api/
The current path, Swal.fire('Good job!', 'You clicked the button!', 'success'), didn't match any of these.

**Comment by guilherme-funchal** [comment]
VM129:1 Uncaught ReferenceError: Swal is not defined
    at <anonymous>:1:1

**Comment by guilherme-funchal** [comment]
Sorry, but not is clear to me where I must install the lib sweetalert2. 

Should I generate the Django container with the library or load it on the base page of the site?

I try to execute in console of Chrome and I get the error:

Uncaught ReferenceError: Swal is not defined

**Comment by Atrox** [comment]
Yeah, so you are missing the sweetalert2 library. Include it in your base template to load it.

**Comment by guilherme-funchal** [comment]
How I do this ?

**Comment by Atrox** [comment]
Put ```<script src="https://cdn.jsdelivr.net/npm/sweetalert2@9"></script>``` into your base template
